File: 1-TwoSum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for(int i =0; i< nums.length - 1; i++){
            int check = target - nums[i];
            if (nums.contains(check) = true){
                return (nums[i] + nums.indexOf[check]);
            }
        } 
    }
}

File: 1-TwoSum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        for i in range(len(nums)):
            answer = target - nums[i]
            for j in range(i,len(nums)):
                if nums[j] == answer:
                    return [i,j]
                
